src/main/kotlin/com/intershop/gradle/resourcelist/task/ResourceListFileTask.kt

```diff
@@ -158,14 +163,17 @@ abstract class ResourceListFileTask
             val java = project.extensions.getByType(JavaPluginExtension::class.java)
             java.sourceSets.all { srcset ->
                 if(srcset.name == sourceSetName) {
-                    (srcset.resources.srcDirs + srcset.allSource.srcDirs).forEach {srcDir ->
-                        val fileSet = project.fileTree(srcDir) {
-                            it.setIncludes(includes)
-                            it.setExcludes(excludes)
-                        }.files
-                        fileSet.forEach {file ->
-                            if(! file.isDirectory) {
-                                setFilePaths.add(file.path.substring(srcDir.path.length + 1))
+                    // search in "resources" only
```

Comment:
Can we rule out that other plugins generate orm or pipelet xml in other source directories?